catalog/core/catalog-core-subscriptionstore/src/main/java/org/codice/ddf/catalog/subscriptionstore/internal/SubscriptionStoreException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.subscriptionstore.internal;
+
+/**
+ * Exception specific to the subscription store.
+ */
+public class SubscriptionStoreException extends RuntimeException {
```

Comment:
I was leaning towards this being a checked exception as well.
Here, https://github.com/codice/ddf/pull/1932/files#diff-9289bc2114fa2e2ec4d523f5478d70d1R73 it is being used as flow control. So there is no reason that at least the exception used there should not be checked. That may mean making a different exception (checked) than this one if this is used too broadly.